var/spack/repos/builtin/packages/pacifica-cli/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PacificaCli(PythonPackage):
+    """Python CLI for Pacifica Core Services"""
+
+    homepage = "https://github.com/pacifica/pacifica-cli/"
```

Comment:
URL should be specified at the package-level so that you can add new versions without adding a new URL every time

Reply:
This is only the homepage for the package, the URL for the actual download is in the version lines?

Reply:
Exactly, the download `url` should be at the same level as the `homepage`. See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi for a new feature I just added that will allow you to replace the `url` with `pypi`.

Reply:
I like the `pypi` option, not sure I can use it yet in our production environment.  :)

Reply:
I have moved the `url` tags to the upper level